sqlmesh/dbt/adapter.py

```diff
@@ -255,15 +259,19 @@ def get_missing_columns(
 
     def create_schema(self, relation: BaseRelation) -> None:
         if relation.schema is not None:
-            self.engine_adapter.create_schema(relation.schema)
+            schema = self._normalize(relation.schema, self.quote_policy.schema)
+            self.engine_adapter.create_schema(schema)
 
     def drop_schema(self, relation: BaseRelation) -> None:
         if relation.schema is not None:
-            self.engine_adapter.drop_schema(relation.schema)
+            schema = self._normalize(relation.schema, self.quote_policy.schema)
+            self.engine_adapter.drop_schema(schema)
```

Comment:
Although the default catalog stuff is not in this PR, you could normalize the catalog here still right? And then do `schema_` on the schema name and catalog name.

Reply:
No you can do this without the catalog PR. Each relation can have a database defined (or catalog in our terms) and therefore you should include that in the call here if one is defined.

Reply:
@eakmanrq changed